src/main/java/com/rocketden/main/dto/game/GameMapper.java

```diff
@@ -51,26 +51,19 @@ public static Game fromRoom(Room room) {
         Game game = new Game();
         game.setRoom(room);
 
+        // Create players and assign colors in order.
+        int index = 0;
         Map<String, Player> players = game.getPlayers();
         for (User user : room.getUsers()) {
-            Player player = playerFromUser(user);
+            Player player = PlayerMapper.playerFromUser(user);
+            player.setColor(Utility.COLOR_LIST.get(index));
             players.put(user.getUserId(), player);
+            index++;
```

Comment:
To prevent out of bounds errors (some users might have the same colors I guess)

Also, do we ever want to randomize the colors? Maybe like create a copy of `COLOR_LIST` and shuffle it would be a quick and easy way - this is beyond the MVP tho so doesn't matter either way

```suggestion
            index = (index + 1) % Utility.COLOR_LIST.size();
```

Reply:
The idea was that since there is a max of ten players in a game (I forgot where I remember this from tbh), the `Utility.COLOR_LIST` field will always have enough colors for all the players. I can still add this though, in case that might change in the future -- though I think it'd be good to eventually have some secondary attribute that we can then pair to make like "100" different combinations, and thus have a unique symbol for each person.

I'm cool with randomizing the colors -- I initially thought of pairing more "opposite" colors across the different stages, like red and blue rather than red and orange and whatnot, but I'm open to whatever for this. I'll implement these after reviewing the remaining PRs, and I can commit this suggestion as well if you think that'd be a good idea.

Reply:
Should I do randomization in this PR @alankbi?

Reply:
Nah I think that can probably be for a later PR